books/kestrel/utilities/ubi-doc.lisp

```diff
@@ -8,6 +8,7 @@
 
 (defxdoc ubi
   :parents (kestrel-utilities history)
+  :code-source "kestrel/utilities/ubi :DIR :SYSTEM"
```

Comment:
The first rendering would just make this text into a link with the appropriate link target to github as the doc links currently do.  If we want to allow lowercase `:dir :system` to mean the same thing when rendering, let me know, that would be easy.
Also, eventually I would like to be able to add a `copy include-book form to clipboard` button to the html xdoc rendering that follows the code-source link.

Reply:
I think it would be good to use lowercase `:dir :system`.

Reply:
The proposal now is that if you say
```
  :code-source "ubi.lisp"
```
it would be a path relative to the current book, but if you say
```
  :code-source ("kestrel/utilities/ubi" :dir :system)
```
it would be relative to the community books.

If "ubi.lisp" exists, then either "ubi" or "ubi.lisp" would find it.

Reply:
Looks good.  Would just` :code-source "ubi"` also work?

Reply:
Oddly, the email github generated with my comment "Looks good" quoted the old example of the `:code-source` option!  What I think actually looks good is Eric's "The proposal now ...".

Reply:
@ericwhitmansmith Yes, `:code-source "ubi"` would also work, and because the `defxdoc` is under `acl2/books`, the code info would render as
```
  Code:  (include-book "kestrel/utilities/ubi" :dir :system)
```